ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ip_nat.schema.yml

```diff
@@ -70,6 +76,7 @@ keys:
                       - str
                     min: 1
                     max: 65535
+                    description: Required when `first_port` is set.
```

Comment:
also we should mention that last port should higher than or equal to first port

Reply:
Done